osu.Game.Rulesets.Tau/Skinning/Legacy/TauLegacySkinTransformer.cs

```diff
@@ -13,18 +13,20 @@ public TauLegacySkinTransformer(ISkin source)
 
         public override Drawable GetDrawableComponent(ISkinComponent component)
         {
-            if (!(component is TauSkinComponent tauComponent))
-                return null;
```

Comment:
It'd probably be best if we left this as is but replace `null` with `base.GetDrawableComponent(component);`

Reply:
I feel it's better sanity wise just in case, it would be better to check the main case, then an alternate if it doesn't apply.

Reply:
Negating the value would only make the code look cleaner and more readable. There's no real "sanity" in this case.

Reply:
fixed in ef816e9